frontend/src/app/components/op-context-menu/wp-context-menu/wp-table-context-menu.directive.ts

```diff
@@ -73,11 +75,15 @@ export class OpWorkPackageContextMenu extends OpContextMenuHandler {
         break;
 
       case 'relation-precedes':
-        this.table.timelineController.startAddRelationPredecessor(this.workPackage);
+        if (this.table) {
```

Comment:
The table specifics would ideally extracted into a subclass. I will fix this as part of the naming refactoring of `RenderedRow`